solidity/dashboard/src/contracts.js

```diff
@@ -125,6 +127,10 @@ const contracts = {
     artifact: LPRewardsTBTCSaddle,
     withDeployBlock: true,
   },
+  [KEEP_TOKEN_GEYSER_CONTRACT_NAME]: {
```

Comment:
We should make sure to deploy these as a part of the local-setup scripts so we don't see local errors.

Still unsure how what that PR would look like 😅. At the very least we should create a GH issue on the local-setup repo when we merge these in.

Reply:
It will work with `local-setup` repo. We deploy `KeepTokenGeyser` contract to the local network in `2_deploy_contracts.js` file, please see https://github.com/keep-network/keep-core/pull/2310/commits/1164b2d3325827d3c3efbd1371e5cfaa7f627f89. The `2_deploy_contracts` will be run via [`truffle migrate`](https://github.com/keep-network/keep-core/blob/master/scripts/install.sh#L35), command ([more here](https://www.trufflesuite.com/docs/truffle/getting-started/running-migrations#command)), which will be triggered by [this script](https://github.com/keep-network/local-setup/blob/master/install-keep-core.sh#L38) in `local-setup` repo.